parameter_files/fates_params_default.cdl

```diff
@@ -1554,6 +1557,9 @@ data:
  fates_turnover_leaf =
   1.5, 4, 1, 1.5, 1, 1, 1.5, 1, 1, 1, 1, 1 ;
 
+ fates_turnover_leaf_ustory =
+  5, 8, 1, 3, 1, 1, 3, 1, 1, 1, 1, 1 ;
```

Comment:
@JessicaNeedham per the discussion with @mpaiao above, should we default these values to the same as fates_turnover_leaf ?

Reply:
Oh yes, I  will update that.